readme.md

```diff
@@ -90,6 +90,37 @@ const FileType = require('file-type');
 })();
 ```
 
+Or from a _tokenizer_:
```

Comment:
I think this should be moved to under the `.fromTokenizer()` docs as it's not something most would need.

Reply:
Done